package.json

```diff
@@ -131,6 +131,7 @@
     "verify-totp",
     "video",
     "video-token",
+    "video-token-server",
```

Comment:
Maybe a philosophical question but should this instead replace the `video-token` app that already exists? Otherwise I'd probably name this with something that makes it easier to differentiate between the two. Looks like the token endpoint is essentially similar with the primary difference being the better setup experience.

Reply:
@dkundel it's a fair question. My understanding is the key difference is this new one gives them a `/token` endpoint where they can pass in the identity/room name as parameters. So developers can use this one as is + then build the client app around it. And then it has the generated passcode and the setup experience changes. 

I know there are token templates for other products that are set up like `video-token`, where developers need to update them to deal with the dynamic identities/etc, so not sure if we want to preserve that consistency or replace it. 

I do think `video-token-server` is pretty descriptive of what the app is. Maybe `video-token-backend`.

Reply:
I think `video-token-backend` is better. Primarily just because `server` tends to be confusing for people considering this is a serverless solution :D